### PR TITLE
Adding Roger to github pages

### PR DESCRIPTION
**Pull request by reyg3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,9 +30,13 @@ We rely on student support and participation across all research projects. Stude
 
 Our current student cohort includes:
 - [Dharma Hoy](https://github.com/dharmahoy)
-  - Major: Statistics and Data Science.
+  - Major: Statistics and Data Science
```
Comment (lboeman) [inline README.md:33]:
```suggestion
  - Major: Statistics and Data Science.
```

File: README.md [hunk 2]
```diff
@@ -30,9 +30,13 @@ We rely on student support and participation across all research projects. Stude
...
   - Minors: Physics, Math, and Information, Science, Technology, and Arts (ISTA)
 
+- [Roger Palmenberg](https://github.com/rogerpalmenberg)
+  - Major:
```
Comment (rogerpalmenberg) [inline README.md]:
```suggestion
  - Major: Statistics and Data Science
```

File: README.md [hunk 3]
```diff
@@ -30,9 +30,13 @@ We rely on student support and participation across all research projects. Stude
 
 Our current student cohort includes:
 - [Dharma Hoy](https://github.com/dharmahoy)
-  - Major: Statistics and Data Science.
+  - Major: Statistics and Data Science
   - Minors: Physics, Math, and Information, Science, Technology, and Arts (ISTA)
 
+- [Roger Palmenberg](https://github.com/rogerpalmenberg)
+  - Major: 
+  - Minors:
```
Comment (rogerpalmenberg) [inline README.md]:
```suggestion

```

**Review by rogerpalmenberg — APPROVED** [review]
No minors:(

File: README.md (hunk 1 above)
Comment (reyg3) [reply README.md:33]: no

**Pull request merged**